<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>WidgetLocation</class>
 <widget class="QWidget" name="WidgetLocation">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>250</width>
    <height>165</height>
   </rect>
  </property>
  <property name="cursor">
   <cursorShape>PointingHandCursor</cursorShape>
  </property>
  <property name="windowTitle">
   <string>Event</string>
  </property>
  <property name="styleSheet">
   <string notr="true">#WidgetLocation {
  background: transparent;
}</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <property name="spacing">
    <number>6</number>
   </property>
   <property name="leftMargin">
    <number>6</number>
   </property>
   <property name="topMargin">
    <number>6</number>
   </property>
   <property name="rightMargin">
    <number>6</number>
   </property>
   <property name="bottomMargin">
    <number>6</number>
   </property>
   <item>
    <widget class="QGroupBox" name="groupBoxMain">
     <property name="styleSheet">
      <string notr="true">#groupBoxMain {
  border-radius: 0px;
  border: 3px solid rgb(131, 147, 167);
  background-color: rgb(80,112,125);
}</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <item>
       <layout class="QVBoxLayout" name="verticalLayout" stretch="0,0,1,0,0">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_2" stretch="1">
          <item>
           <widget class="QLabel" name="labelHead">
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>24</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>16777215</width>
              <height>24</height>
             </size>
            </property>
            <property name="font">
             <font>
              <family>Roboto</family>
              <pointsize>10</pointsize>
              <weight>75</weight>
              <bold>true</bold>
             </font>
            </property>
            <property name="styleSheet">
             <string notr="true">color: white;</string>
            </property>
            <property name="text">
             <string>Location Name</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QLabel" name="labelLineHeader">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>2</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>16777215</width>
            <height>2</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true">QLabel {
  background-color: rgb(131,147, 167);
}</string>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout" stretch="1,0">
          <item>
           <widget class="QLabel" name="labelDescription">
            <property name="font">
             <font>
              <family>Roboto</family>
              <pointsize>9</pointsize>
             </font>
            </property>
            <property name="styleSheet">
             <string notr="true">color: white;</string>
            </property>
            <property name="text">
             <string>Description</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
            </property>
            <property name="wordWrap">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="labelPhoto">
            <property name="minimumSize">
             <size>
              <width>64</width>
              <height>64</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>64</width>
              <height>64</height>
             </size>
            </property>
            <property name="scaledContents">
             <bool>true</bool>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QLabel" name="labelLineFooter">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>1</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>16777215</width>
            <height>1</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true">QLabel {
  background-color: rgb(131,147, 167);
}</string>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayoutFooter">
          <property name="spacing">
           <number>8</number>
          </property>
          <item>
           <widget class="QWidget" name="widgetVotes" native="true">
            <property name="toolTip">
             <string>No votes</string>
            </property>
            <property name="styleSheet">
             <string notr="true">#widgetVotes {
  border-radius: 2px;
  border: 1px solid rgb(131, 147, 167);
  background-color: rgb(131,147, 167);
}</string>
            </property>
            <layout class="QHBoxLayout" name="horizontalLayout_4">
             <property name="leftMargin">
              <number>6</number>
             </property>
             <property name="topMargin">
              <number>3</number>
             </property>
             <property name="rightMargin">
              <number>6</number>
             </property>
             <property name="bottomMargin">
              <number>3</number>
             </property>
             <item>
              <widget class="QLabel" name="label">
               <property name="font">
                <font>
                 <family>Roboto</family>
                 <pointsize>9</pointsize>
                </font>
               </property>
               <property name="styleSheet">
                <string notr="true">color: rgb(238, 238, 236);</string>
               </property>
               <property name="text">
                <string>Votes</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLabel" name="labelVotes">
               <property name="font">
                <font>
                 <family>Roboto</family>
                 <pointsize>9</pointsize>
                </font>
               </property>
               <property name="styleSheet">
                <string notr="true">color: rgb(238, 238, 236);</string>
               </property>
               <property name="text">
                <string>0</string>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QPushButton" name="pushButtonVoteDown">
            <property name="minimumSize">
             <size>
              <width>20</width>
              <height>20</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>20</width>
              <height>20</height>
             </size>
            </property>
            <property name="focusPolicy">
             <enum>Qt::NoFocus</enum>
            </property>
            <property name="toolTip">
             <string>Vote down</string>
            </property>
            <property name="styleSheet">
             <string notr="true">QPushButton {
  background: transparent;
  border-image: url(:/icon-thumbdown.png);
}
QPushButton:hover {
  background-color: white;
  border: 1px solid rgb(115, 210, 22);
  border-radius:  4px;
}</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="pushButtonVoteUp">
            <property name="minimumSize">
             <size>
              <width>20</width>
              <height>20</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>20</width>
              <height>20</height>
             </size>
            </property>
            <property name="focusPolicy">
             <enum>Qt::NoFocus</enum>
            </property>
            <property name="toolTip">
             <string>Vote up</string>
            </property>
            <property name="styleSheet">
             <string notr="true">QPushButton {
  background: transparent;
  border-image: url(:/icon-thumbup.png);
}
QPushButton:hover {
  background-color: white;
  border: 1px solid rgb(115, 210, 22);
  border-radius:  4px;
}</string>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer_2">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeType">
             <enum>QSizePolicy::Fixed</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>12</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QPushButton" name="pushButtonInfo">
            <property name="minimumSize">
             <size>
              <width>20</width>
              <height>20</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>20</width>
              <height>20</height>
             </size>
            </property>
            <property name="focusPolicy">
             <enum>Qt::NoFocus</enum>
            </property>
            <property name="toolTip">
             <string>Info</string>
            </property>
            <property name="styleSheet">
             <string notr="true">QPushButton {
  background: transparent;
  border-image: url(:/icon-info.png);
}
QPushButton:hover {
  background-color: white;
  border: 1px solid rgb(115, 210, 22);
  border-radius:  4px;
}</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="pushButtonSettings">
            <property name="minimumSize">
             <size>
              <width>20</width>
              <height>20</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>20</width>
              <height>20</height>
             </size>
            </property>
            <property name="focusPolicy">
             <enum>Qt::NoFocus</enum>
            </property>
            <property name="toolTip">
             <string>Settings</string>
            </property>
            <property name="styleSheet">
             <string notr="true">QPushButton {
  background: transparent;
  border-image: url(:/icon-settings.png);
}
QPushButton:hover {
  background-color: white;
  border: 1px solid rgb(115, 210, 22);
  border-radius:  4px;
}</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>pushButtonVoteDown</sender>
   <signal>clicked()</signal>
   <receiver>WidgetLocation</receiver>
   <slot>onBtnVoteDownClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>113</x>
     <y>135</y>
    </hint>
    <hint type="destinationlabel">
     <x>245</x>
     <y>161</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButtonVoteUp</sender>
   <signal>clicked()</signal>
   <receiver>WidgetLocation</receiver>
   <slot>onBtnVoteUpClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>145</x>
     <y>131</y>
    </hint>
    <hint type="destinationlabel">
     <x>249</x>
     <y>121</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButtonSettings</sender>
   <signal>clicked()</signal>
   <receiver>WidgetLocation</receiver>
   <slot>onBtnSettingsClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>223</x>
     <y>134</y>
    </hint>
    <hint type="destinationlabel">
     <x>250</x>
     <y>86</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>onBtnVoteDownClicked()</slot>
  <slot>onBtnVoteUpClicked()</slot>
  <slot>onBtnSettingsClicked()</slot>
 </slots>
</ui>
